jobs/integration/test_service_endpoints.py

```diff
@@ -2,6 +2,8 @@
 import yaml
 from .utils import retry_async_with_timeout, juju_run
 
+SVC_PORT = 50000
```

Comment:
toss this port number in everywhere it's mentioned in here.  I see some on lines 49 and 59